src/controllers/signAccountOp/signAccountOp.ts

```diff
@@ -480,6 +480,12 @@ export class SignAccountOpController extends EventEmitter {
       this.estimation.erc4337GasLimits.gasPrice = bundlerGasPrices
     }
 
+    const shouldUpdateAccountOpCalls =
+      accountOp && accountOp.calls.length !== this.accountOp.calls.length
+    if (shouldUpdateAccountOpCalls) {
+      this.accountOp.calls = accountOp.calls
```

Comment:
this.accountOp = accountOp

Reply:
I used the spread operator, just in case the incoming one has a prop missing in the current one, in https://github.com/AmbireTech/ambire-common/pull/1011/commits/7ef3abd1a640557fbfe11e8cd3c328c911e3f091